### PR TITLE
Add nil check

### LINKED ISSUE #6831: datanode crashed in meta service
#### Steps/Code to reproduce:
test cases in CI

#### Expected result:
tests passed

#### Actual results:
https://ci.milvus.io:18080/jenkins/blue/organizations/jenkins/milvus-ha-ci/detail/PR-6819/2/pipeline/106/

#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:

### PR DESCRIPTION
**Pull request by bigsheeper**
Signed-off-by: bigsheeper <yihao.dai@zilliz.com>

Resolves: #6831

### CODE REVIEW

File: internal/datanode/meta_service.go [hunk 1]
```diff
@@ -55,8 +55,8 @@ func (mService *metaService) getCollectionSchema(ctx context.Context, collID Uni
 	}
 
 	response, err := mService.rootCoord.DescribeCollection(ctx, req)
-	if response.Status.ErrorCode != commonpb.ErrorCode_Success {
-		return nil, fmt.Errorf("Describe collection %v from rootcoord wrong: %s", collID, err.Error())
+	if response != nil && response.Status.ErrorCode != commonpb.ErrorCode_Success {
```
Comment (XuanYang-cn) [inline internal/datanode/meta_service.go:58]: It's better to use grpc functions
```go
if response.GetStatus().GetErrorCode() != commonpb.ErrorCode_Success {
    return nil, fmt.Errorf("Describe collection %v from rootcoord wrong: %s", collID, response.GetStatus().GetReason())
}
```

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: internal/datanode/meta_service.go (hunk 1 above)
Comment (bigsheeper) [reply internal/datanode/meta_service.go:58]: 👌

**Pull request merged**